tests/test_api.py

```diff
@@ -94,13 +95,13 @@ def test_phone_verification_with_incorrect_payload(client, backend):
         }
         response = client.json.post(url, data=data)
         response_data = json.loads(json.dumps(response.data))
-        if backend.get("BACKEND").split("phone_verify.backends.")[1] in backends:
+
+        backend_cls = _get_backend_cls(backend.get("BACKEND"))
```

Comment:
Let's just pass the backend to the function as suggested in the previous comment?